src/Avalonia.Controls/Window.cs

```diff
@@ -104,6 +104,9 @@ public class Window : WindowBase, IStyleable, IFocusScope, ILayoutRoot, INameSco
         public static readonly StyledProperty<bool> CanResizeProperty =
             AvaloniaProperty.Register<Window, bool>(nameof(CanResize), true);
 
+        public static readonly StyledProperty<bool> TopmostProperty =
```

Comment:
This property needs to be moved to WindowBase since TopMost can be set on PopupRoot also.